### PR TITLE
update install.md to include go lang pre-reqs when deploying

### PR DESCRIPTION
**Pull request by inderpaltiwana**
Updating the install Readme to include go lang dependency when deploying, and attempting to run the make deploy command.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 * [kustomize](https://github.com/kubernetes-sigs/kustomize#kustomize)
 * [oc](https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/)
+* [go](https://github.com/golang/go)
```
Comment (LalatenduMohanty) [inline docs/install.md]: We should add go version here i.e. s/go/go version 1.12/

Comment (inderpaltiwana) [reply docs/install.md]: updated the readme go prerequisite to include go version 1.12 as suggested

**Pull request closed without merging**